### DISCUSSION TITLE
Scanning the ticket does not add the scanned#

### DISCUSSION THREAD

**Discussion by robhiddinga** [liedekef/events-made-easy#376] [Q&A] [answered]
In my PDF I send the #_ATTENDANCE_QRCODE but that QR  can be scanned endless times,
And same goes for #_QRCODE
On the scan resultscreen the max-number is correct and corresponds to the number of bought ticktes,
But the times_sanned (aantal keer gescand=0) always is and stays zero
I do keep track of the attendees and the buyer is added to that list at every scan 

Anyone encountered this behaviour before?  
I can't find anything in forum or documentation

**Comment by liedekef** [comment, marked as answer]
This is not reproducable (works fine here). Are you sure you have the needed rights to scan? In any scan the count goes up fine here.
Also make sure no caching plugin can interfere ...

**Reply by liedekef** [reply]
no reaction, considering solved.
